users/validators.py

```diff
@@ -4,14 +4,15 @@
 MAX_YEARS_OF_EXPERIENCE = 100
 
 
-class ProfileValidator(object):
+class ProfileValidator:
     @staticmethod
     def validate_experience(rq_data):
         experience = rq_data.get('experience')
 
-        if isinstance(experience, int) and (
-                experience > MAX_YEARS_OF_EXPERIENCE or
-                experience < MIN_YEARS_OF_EXPERIENCE):
+        if not isinstance(experience, int):
+            raise forms.ValidationError('Enter correct data')
```

Comment:
User might be confused by this message. I  propose to provide a bit more specific feedback. E.g. something like this one:
"Only integer values are supported".